proto/ibc/lightclients/ethmultisig/v1/ethmultisig.proto

```diff
@@ -2,7 +2,9 @@ syntax = "proto3";
 
 option go_package = "github.com/datachainlab/ibc-ethmultisig-client/modules/light-clients/xx-ethmultisig/types";
 
+import "solidity-protobuf-extensions.proto";
```

Comment:
I wonder if this is required

Reply:
The following code added import for solidity-protobuf-extensions.proto, so I used this as a reference.
https://github.com/hyperledger-labs/yui-ibc-solidity/blob/main/proto/client/ibft2/IBFT2.proto#L5
https://github.com/hyperledger-labs/yui-ibc-solidity/blob/main/proto/client/mock/MockClient.proto#L5

Reply:
I think it's because they use an option `.solidity.file_options`.